### PR TITLE
Fix typos and grammar in BigStep

### PR DESCRIPTION
**Pull request by abt8601**

### CODE REVIEW

File: src/plfa/part2/BigStep.lagda.md [hunk 1]
```diff
@@ -308,7 +308,7 @@ to consider.
 
 * Case `⇓-var`.
   So we have `γ x ≡ clos L δ` and `δ ⊢ L ⇓ V`.
-  We need to show that `subst σ x —↠ N` and `V ≈ N` for some `N`.
+  We need to show that ``subst σ (` x) —↠ N`` and `V ≈ N` for some `N`.
```
Comment (abt8601) [inline src/plfa/part2/BigStep.lagda.md:311]: Or do you prefer `σ x —↠ N`?

**Comment by wadler** [comment]
Leaving Jeremy to comment on this one!

**Review by jsiek — APPROVED** [review]
These changes look good to me. Thank you!

File: src/plfa/part2/BigStep.lagda.md (hunk 1 above)
Comment (jsiek) [reply src/plfa/part2/BigStep.lagda.md:311]: I lean towards keeping the `subst`.

**Pull request merged**